TestProjects/Forms/Example001CSharp/Example.UWP/Example.UWP.csproj

```diff
@@ -139,14 +139,6 @@
       <Project>{0797D39C-C59B-4FE2-BB16-AB6FA0119C69}</Project>
       <Name>MvvmCross.Forms</Name>
     </ProjectReference>
-    <ProjectReference Include="..\..\..\..\MvvmCross\Binding\UWP\MvvmCross.Binding.Uwp.csproj">
-      <Project>{637533ee-756b-4817-94b2-7c1e56132208}</Project>
-      <Name>MvvmCross.Binding.Uwp</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\..\..\MvvmCross\Core\Binding\MvvmCross.Binding.csproj">
-      <Project>{64dcd397-9019-41e8-a928-e5f5c5df185b}</Project>
-      <Name>MvvmCross.Binding</Name>
-    </ProjectReference>
```

Comment:
I'm afraid the removal of the 2 project references is a mistake. It will lead to compiling error:
```
\MvvmCross\TestProjects\Forms\Example001CSharp\Example.UWP\Example.UWP.csproj : XamlCompiler error WMC1006: Cannot resolve Assembly or Windows Metadata file 'MvvmCross.Binding.dll'
Microsoft.Windows.UI.Xaml.Common.targets(354,5): Xaml Internal Error error WMC9999: Type universe cannot resolve assembly: MvvmCross.Binding, Version=5.0.3.0, Culture=neutral, PublicKeyToken=null.
```
I created PR #2007 , please help to review.